starter-code/src/component/CountryDetail.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Countries from "../countries.json";
+import { Link, Route } from "react-router-dom";
+
+const CountryDetail = props => {
+  // Recibo los parámetros que hay en la URL desde App.js vía el <ROUTE>
+  const { countryCode } = props.match.params;
+  console.log({ countryCode });
+
+  // Funcion para encontrar el CCA3 según el CountryCode que recibimos
+  const handeldCountry = countryCCA => {
+    const result = Countries.find(country => country.cca3 === countryCCA);
+    return result;
+  };
+
+  // LLamamos a la función que hay justo encima.
+  const countryToDisplay = handeldCountry(countryCode);
```

Comment:
Aquí podéis hacer un `destructuring` para que después el manejo de la variable sea más cómodo.
```
const { name, area, capital, borders } = handeldCountry(countryCode);
```